ocpi-toolkit-2.2.1/src/main/kotlin/com/izivia/ocpi/toolkit/modules/credentials/services/CredentialsClientService.kt

```diff
@@ -36,18 +37,19 @@ open class CredentialsClientService(
     private val clientVersionsRepository: VersionsRepository,
     private val clientCredentialsRoleRepository: CredentialsRoleRepository,
     private val serverVersionsEndpointUrl: String,
```

Comment:
`serverVersionsEndpointUrl` is not required anymore because the information is held by the partner that we can retrieve using `partnerId`